user/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+class Users(models.Model):
+        email            = models.EmailField(
+                max_length = 50,
+                #validators = [validate_email],
+                #unique     = True,                       
+        )
+        name           = models.CharField(max_length=50)
+        password        = models.CharField(
+                max_length = 300,
+                #validators = [validate_password],
+        )
+        phone_number    = models.CharField(max_length=50)
+        created_at      = models.DateTimeField(auto_now_add = True)
+        updated_at      = models.DateTimeField(auto_now = True)
```

Comment:
```suggestion
        phone_number    = models.CharField(max_length = 50)
        created_at      = models.DateTimeField(auto_now_add = True)
        updated_at      = models.DateTimeField(auto_now = True)
```
붙이셔도 되고 띄우셔도 됩니다.
하지만 한가지 스타일로 통일하시는 것을 추천드립니다 👍